ocrd_utils/ocrd_utils/package_resources.py

```diff
@@ -0,0 +1,45 @@
+import atexit
+from contextlib import ExitStack
+from pathlib import Path
+
+try:
+    from importlib.resources import path, read_binary
+except ImportError:
+    from importlib_resources import path, read_binary  # type: ignore
+
+
+_file_manager = ExitStack()
+atexit.register(_file_manager.close)
+
+
+def resource_filename(package: str, resource: str) -> Path:
+    """
+    Reimplementation of the function with the same name from pkg_resources
+
+    Using importlib for better performance
+
+    package : str
+        The package from where to start looking for resource (often __name__)
+    resource : str
+        The resource to look up
+    """
+    parent_package = package.rsplit('.',1)[0]
+    return _file_manager.enter_context(path(parent_package, resource))
```

Comment:
Why enter the context of an ExitStack here?

Reply:
This is adapted from a recipe in the[importlib_resources migration guide](https://importlib-resources.readthedocs.io/en/latest/migration.html#pkg-resources-resource-filename). The returned file name could be a temporary file (e.g. extracted from a zip). The importlib_resources-functions return a context manager to manage the lifecycle of that file. To mimic the semantics of resource_filename, I had to get rid of the context manager, so I used the ExitStack recipe to have the file available for the whole lifetime of the process and cleaned up at exit. 
The migration explains it better than me ;)

Reply:
I see, so this ensures that paths are retained even when the package is accessed in a zipped format. Not 100% sure I understand when this would happen. I'll add a comment referencing the migration guide.